lib/config.js

```diff
@@ -74,6 +74,9 @@ module.exports = (function () {
           enabled: false,
           clientId: "replace me with the real value",
           clientSecret: "replace me with the real value",
+          authorizationURL: "leave me empty to https://github.com/login/oauth/authorize",
+          tokenURL: "leave me empty to https://github.com/login/oauth/access_token",
+          userProfileURL: "leave me empty to https://api.github.com/user",
```

Comment:
I think that for the default values, either you leave it really empty or you use those default URLs. The strings as they are, are a bit confusing, IMHO.